src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  font-weight: normal;
+  color: $main-accent;
+
+  width: 200px;
+  margin: 0;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
+
+  &__product-img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    font-size: 12px;
+    line-height: $line-height-big;
+    font-weight: $weight-medium;
+    margin-bottom: 4px;
+  }
+
+  &__product-id {
+    margin-bottom: 16px;
+    line-height: $line-height-small;
+    font-size: 10px;
+    color: $secondary;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    line-height: 16px;
+  }
+
+  &__reviews-count {
+    font-size: 10px;
+    align-items: center;
+    line-height: $line-height-small;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      color: $secondary;
+      line-height: $line-height-big;
+    }
+
+    &-value {
+      font-weight: $weight-bold;
+      font-size: 16px;
+      line-height: $line-height-big;
+    }
+  }
+
+  &__buy-button {
+    display: inline-block;
+    padding: 11px 0;
+    width: 166px;
```

Comment:
do not hardcode elements' width unless you have to. here the button's width needs to be 100% of the parent. simply use display block without having to set the width